language/tools/move-cli/README.md

```diff
@@ -24,21 +31,82 @@ fun main(account: &signer) {
 }
 ```
 
-Place this in a file named `script.move` and try
+Place this in a file named `debug_script.move` and try
+
 ```
-λ move run script.move --signers 0xf
-[debug] (&) { 0000000000000000000000000000000f }
+λ move run debug_script.move --signers 0xf
+[debug] (&) { 0000000000000000000000000000000F }
 ```
 
-The `--signers 0xf` part indicates which account address(es) have signed off on the script. Omitting `--signers` or passing multiple signers to this single-`signer` script will trigger a type error.
-
-## Adding new modules via `src`
-
-By default, the CLI compiles and includes all files from the Move standard library and Libra Framework. New modules can be added to the `src` directory (or a directory of your choosing specified via `--source-dir`. The `move run` command will compile and publish each module source file in this directory before running the given script.
+The `--signers 0xf` argument indicates which account address(es) have signed
+off on the script. Omitting `--signers` or passing multiple signers to this
+single-`signer` script will trigger a type error.
+
+## Modes and adding new modules
```

Comment:
Modes seem like a more advanced topic to me, particularly given that we've chosen a good `default` mode that most users won't need to change... Perhaps move this into the "Using CLI with the Libra Framework" section at the end?

Reply:
Sounds like a good idea to me! I'll move down.